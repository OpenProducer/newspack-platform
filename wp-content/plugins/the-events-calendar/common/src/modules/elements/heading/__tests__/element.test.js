import React from 'react';

import Heading from '../element';

describe( '<Heading>', () => {
	test( '<h1>', () => {
		const component = renderer.create(
			<Heading level={ 1 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( '<h2>', () => {
		const component = renderer.create(
			<Heading level={ 2 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( '<h3>', () => {
		const component = renderer.create(
			<Heading level={ 3 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( '<h4>', () => {
		const component = renderer.create(
			<Heading level={ 4 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( '<h5>', () => {
		const component = renderer.create(
			<Heading level={ 5 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( '<h6>', () => {
		const component = renderer.create(
			<Heading level={ 6 }>Modern Tribe</Heading>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
