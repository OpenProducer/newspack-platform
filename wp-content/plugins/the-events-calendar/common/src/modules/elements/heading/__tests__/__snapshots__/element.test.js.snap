// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading> <h1> 1`] = `
<h1
  className="tribe-editor__heading tribe-editor__heading--h1"
>
  Modern Tribe
</h1>
`;

exports[`<Heading> <h2> 1`] = `
<h2
  className="tribe-editor__heading tribe-editor__heading--h2"
>
  Modern Tribe
</h2>
`;

exports[`<Heading> <h3> 1`] = `
<h3
  className="tribe-editor__heading tribe-editor__heading--h3"
>
  Modern Tribe
</h3>
`;

exports[`<Heading> <h4> 1`] = `
<h4
  className="tribe-editor__heading tribe-editor__heading--h4"
>
  Modern Tribe
</h4>
`;

exports[`<Heading> <h5> 1`] = `
<h5
  className="tribe-editor__heading tribe-editor__heading--h5"
>
  Modern Tribe
</h5>
`;

exports[`<Heading> <h6> 1`] = `
<h6
  className="tribe-editor__heading tribe-editor__heading--h6"
>
  Modern Tribe
</h6>
`;
